appveyor.yml

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
Is this from an incomplete merge?

Reply:
I don't know how to use appveyor or travis yet so let me know if certain things are missing from these files.